programs/clearing_house/src/controller/position.rs

```diff
@@ -545,14 +672,20 @@ mod test {
         let mut market = Market {
             amm: AMM {
                 cumulative_funding_rate_long: 1,
+                sqrt_k: 1,
                 ..AMM::default()
             },
             open_interest: 0,
-            ..Market::default()
+            ..Market::default_test()
         };
 
-        let pnl = update_position_and_market(&mut existing_position, &mut market, &position_delta)
-            .unwrap();
+        let pnl = update_user_and_market_position(
```

Comment:
why did we only change this test? this should be set back to `update_position_and_market` and we should have a separate set of tests for `update_user_and_market_position`

Reply:
the original tests tested update_position_and_market and its change to net_baa - since update_position_and_market doesnt change net_baa (update_user_and_market_position does) and there are no lps - the two functions are equivalent; the testing coverage is the same